src/routes/jira/server/jira-server-url-post.ts

```diff
@@ -12,15 +12,18 @@ interface GheServerUrlErrors {
 	[key: string | number]: MessageAndCode;
 }
 
+const GHE_ERROR_ENOTFOUND = {
+	errorCode: "GHE_ERROR_ENOTFOUND",
+	message: "Request to URL failed"
+};
+
 export const gheServerUrlErrors: GheServerUrlErrors = {
 	invalidUrl: {
 		errorCode: "GHE_ERROR_INVALID_URL",
 		message: "Invalid URL"
 	},
-	ENOTFOUND: {
-		errorCode: "GHE_ERROR_ENOTFOUND",
-		message: "Request to URL failed"
-	},
+	ENOTFOUND: GHE_ERROR_ENOTFOUND,
+	403: GHE_ERROR_ENOTFOUND,
```

Comment:
Could be a bit weird, since 404 is NOT_FOUND, while 403 is Forbidden.

Reply:
And is it for UX purpose that we need to tell the difference of 403 and ENOTFOUND?